### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by SimoTod**
Can you have a quick look and see if it makes sense to you please? Thanks

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,19 +16,23 @@
     "build": "rollup -c",
     "lint": "eslint src",
     "lint-fix": "eslint src --fix",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "pretest": "http-server &",
+    "test": "npx cypress run || :",
```
Comment (HugoDF) [inline package.json:20]: I don't think you need `npx` inside of npm script definitions
```suggestion
    "test": "cypress run || :",
```

File: tests/integration/for.spec.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/* global describe, it, cy, expect */
+
+describe('x-for directives', () => {
+  it('should not add duplicate items when navigating away and back to the page', () => {
+    cy.visit('http://127.0.0.1:8080/tests/res/for/index.html')
```
Comment (HugoDF) [inline tests/integration/for.spec.js:5]: we could use a `baseURL` and then could do `visit(/tests/res/for/index.html)`

File: package.json (hunk 1 above)
Comment (SimoTod) [reply package.json:20]: Thanks

File: tests/integration/for.spec.js (hunk 2 above)
Comment (SimoTod) [reply tests/integration/for.spec.js]: I've changed it but the assertion still need the full url so it hasn't improved drastically.

**Pull request merged**